core/client/pom.xml

```diff
@@ -256,7 +221,7 @@
               <relocations>
                 <relocation>
                   <pattern>org.apache.thrift</pattern>
-                  <shadedPattern>shaded.alluxio.org.apache.thrift</shadedPattern>
+                  <shadedPattern>${project.artifactId}.org.apache.thrift</shadedPattern>
```

Comment:
my concern is that hypen is actually not considered valid for java package naming. 
http://stackoverflow.com/questions/3179216/what-is-the-convention-for-word-separator-in-java-package-names
even it works for a particular environment, I am not sure it will introduce troubles for other JVMs